vm.go

```diff
@@ -225,6 +261,23 @@ func (vm *VM) EvaluateSnippetMulti(filename string, snippet string) (files map[s
 	return
 }
 
+// EvaluateSnippetDependencies evaluates a string containing Jsonnet code returning an array of dependencies.
+//
+// The filename parameter is only used for error messages.
+func (vm *VM) EvaluateSnippetDependencies(filename string, snippet string) (dependencies []string, formattedErr error) {
+	node, err := SnippetToAST(filename, snippet)
+	if err != nil {
+		return nil, err
+	}
+	var stackTrace []traceFrame
+	err = vm.evaluateDependencies(filename, parser.Children(node), &dependencies, &stackTrace, 0)
```

Comment:
What do you think about sorting the dependencies before returning them (with `sort.Strings`)?

This way we could make the output more stable (removing one dependency wouldn't change the relative order of anything that remains).

Reply:
Makes sense. Done.